eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala

```diff
@@ -74,11 +74,15 @@ object ChannelRelayer {
           Behaviors.receiveMessage {
             case Relay(channelRelayPacket) =>
               val relayId = UUID.randomUUID()
-              val channels: Map[ShortChannelId, Relayer.OutgoingChannel] = channelUpdates.get(channelRelayPacket.payload.outgoingChannelId) match {
-                case Some(channel) => node2channels.get(channel.nextNodeId).map(channelUpdates).map(c => c.channelUpdate.shortChannelId -> c).toMap
+              val nextNodeId_opt: Option[PublicKey] = channelUpdates.get(channelRelayPacket.payload.outgoingChannelId) match {
+                case Some(channel) => Some(channel.nextNodeId)
+                case None => None
+              }
```

Comment:
nit:
```suggestion
              val nextNodeId_opt: Option[PublicKey] = channelUpdates.get(channelRelayPacket.payload.outgoingChannelId).map(_.nextNodeId)
```

Reply:
The format I went with allows for easier merge in feature branches.

Reply:
Ok, got it!